cypress/integration/visitorCanChooseArticlesByCategories.feature.js

```diff
@@ -0,0 +1,35 @@
+describe("Visitor can choose articles by categories", () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v1/articles",
+      response: "fixture:articles_index.json",
+    });
+    cy.visit("/");
+  });
+  context("", () => {
+    it("visitor can see different categories in the header", () => {
+      cy.get("[data-cy='news']").should("contain", "News");
+      cy.get("[data-cy='sports']").should("contain", "Sports");
+      cy.get("[data-cy='politics']").should("contain", "Politics");
+    });
+  });
+
+  context("successfully - News", () => {
+    beforeEach(() => {
+      cy.server();
+      cy.route({
```

Comment:
I think this is a false positive, this should not work with your current implementation